pipeline/07-export.R

```diff
@@ -244,6 +341,13 @@ for (town in unique(assessment_pin_prepped$township_code)) {
   writeFormula(
     wb, pin_sheet_name,
     assessment_pin_filtered$meta_pin,
+    startCol = 1,
```

Comment:
We need to specify `startCol` in these formulas now, since there are two of them; otherwise, the second formula clobbers the first one (which is confusing API behavior, but it's what we have to work with).